### PR TITLE
Health bot tests

### PR DESCRIPTION
**Pull request by mcgallan**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/healthbot/Azure.ResourceManager.HealthBot/tests/SessionRecords/HealthBotTests/HealthBotApiTests.json [hunk 1]
```diff
@@ -0,0 +1,2097 @@
+{
```
Comment (ArthurMa1978) [inline sdk/healthbot/Azure.ResourceManager.HealthBot/tests/SessionRecords/HealthBotTests/HealthBotApiTests.json:1]: Please move to TestProxy.

File: sdk/healthbot/Azure.ResourceManager.HealthBot/tests/TestCase/HealthBotTests.cs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            Assert.AreEqual(data1.Name, data2.Name);
+            Assert.AreEqual(data1.Id, data2.Id);
+            Assert.AreEqual(data1.Location, data2.Location);
+        }
+
+        [RecordedTest]
+        public async Task HealthBotApiTests()
+        {
+            //1.CreateOrUpdate
+            var collection = await GetHealthBotCollection();
```
Comment (ArthurMa1978) [inline sdk/healthbot/Azure.ResourceManager.HealthBot/tests/TestCase/HealthBotTests.cs:42]: Please refer the generated samples to add more tests for all resources.

File: sdk/healthbot/Azure.ResourceManager.HealthBot/tests/SessionRecords/HealthBotTests/HealthBotApiTests.json (hunk 1 above)
Comment (mcgallan) [reply sdk/healthbot/Azure.ResourceManager.HealthBot/tests/SessionRecords/HealthBotTests/HealthBotApiTests.json]: fixed

File: sdk/healthbot/Azure.ResourceManager.HealthBot/tests/TestCase/HealthBotTests.cs (hunk 2 above)
Comment (mcgallan) [reply sdk/healthbot/Azure.ResourceManager.HealthBot/tests/TestCase/HealthBotTests.cs:45]: fixed

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**